node/network/dispute-distribution/src/metrics.rs

```diff
@@ -106,6 +114,13 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
+			time_dispute_request: prometheus::register(
+				prometheus::Histogram::with_opts(prometheus::HistogramOpts::new(
+					"parachain_dispute_distribution_time_dispute_request",
+					"Time needed for dispute votes to get confirmed/fail getting transmitted.",
+				))?,
+				registry,
+			)?,
```

Comment:
the default buckets are https://docs.rs/prometheus/0.12.0/src/prometheus/histogram.rs.html#25-27